DataFormats/ForwardDetId/interface/MTDChannelIdentifier.h

```diff
@@ -0,0 +1,77 @@
+#ifndef DATAFORMATS_MTDCHANMNELIDENTIFIER_H
```

Comment:
I just noticed the typo here: CHANMNEL -> CHANNEL
not too important, but could be fixed along with the other issue just posted